### PR TITLE
[PubsubPluginSink] Drop requests greater than MAX_INPUT_METRICS

### PR DESCRIPTION
**Pull request by lmuhlha**

### CODE REVIEW

File: modules/pubsub/src/main/java/com/spotify/ffwd/pubsub/PubsubPluginSink.java [hunk 1]
```diff
@@ -119,22 +122,32 @@ public void onSuccess(String messageId) {
...
+              m.size());
+          int times = (int) Math.ceil(m.size() / MAX_BATCH_SIZE_BYTES);
+          List<List<Metric>> collections =
+              Lists.partition(new ArrayList<>(metrics), metrics.size() / times);
+          for (List<Metric> l : collections) {
+            try {
+              final ByteString mResize = ByteString.copyFrom(serializer.serialize(l, writeCache));
+              publishPubSub(mResize);
+            } catch (Exception e) {
+              logger.error("Failed to serialize batch of metrics: ", e);
```
Comment (samfadrigalan) [inline modules/pubsub/src/main/java/com/spotify/ffwd/pubsub/PubsubPluginSink.java:139]: should we throw an HttpException where we're catching these exceptions?

Comment (hexedpackets) [reply modules/pubsub/src/main/java/com/spotify/ffwd/pubsub/PubsubPluginSink.java:139]: I don't think this is in the path of HTTP requests, the input plugins are done once ffwd has buffered the metrics internally.

Comment (lmuhlha) [reply modules/pubsub/src/main/java/com/spotify/ffwd/pubsub/PubsubPluginSink.java:139]: I don't think I can throw an exception at that point, we'd need to figure out the size sooner somewhere upstream

**Review by hexedpackets — APPROVED** [review]

File: modules/pubsub/src/main/java/com/spotify/ffwd/pubsub/PubsubPluginSink.java (hunk 1 above)
Comment (lmuhlha) [reply modules/pubsub/src/main/java/com/spotify/ffwd/pubsub/PubsubPluginSink.java:139]: lol hey william

**Pull request merged**